newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -8,10 +8,16 @@ import BehaviorsEditor from '../BehaviorsEditor';
 import { Tabs, Tab } from 'material-ui/Tabs';
 import { withSerializableObject } from '../Utils/SerializableObjectEditorContainer';
 
+import SemiControlledTextField from '../UI/SemiControlledTextField';
+
 const styles = {
   titleContainer: {
     padding: 0,
   },
+  objectTitle: {
+    left: "10px",
+    width:"available",
```

Comment:
width should not be needed, I think you can use `fullWidth` prop on the SemiControlledTextField.

Reply:
The fullWidth prop pushed it to another row, so instead I used a div container with css style inspired from another place  in the editor